src/template.js

```diff
@@ -1,22 +1,61 @@
 
     <script>
-      window.enableWebSocket = <%- escapeJson(enableWebSocket) %>;
+      window.enableWebSocket = ${escapeJson(enableWebSocket)};
     </script>
-    <%- include('script', { filename: 'viewer.js' }) %>
+    ${getScript('viewer.js', mode)}
   </head>
 
   <body>
     <div id="app"></div>
     <script>
-      window.chartData = <%- escapeJson(chartData) %>;
-      window.defaultSizes = <%- escapeJson(defaultSizes) %>;
+      window.chartData = ${escapeJson(chartData)};
+      window.defaultSizes = ${escapeJson(defaultSizes)};
     </script>
   </body>
-</html>
+</html>`;
+}
+
+exports.renderViewer = renderViewer;
```

Comment:
Let's move it above `escapeJson` function declaration please so exported members become clearly visible.

Reply:
Done